test/beast/http/fields.cpp

```diff
@@ -24,6 +24,9 @@ namespace http {
 class fields_test : public beast::unit_test::suite
 {
 public:
+    static constexpr std::size_t max_static_buffer =
+        sizeof(http::detail::temporary_buffer<std::allocator<char>>);
```

Comment:
This looks like a cheat

Reply:
The object can't hide `sizeof + 1` bytes in it, so it's good enough, without requiring a definition visible to user code.